test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/NodeEndToEndTests.cs

```diff
@@ -885,13 +885,34 @@ await ApiHubTestHelper.AssertTextUpdatedAsync(
         
 #endif
 
-        public class TestFixture : EndToEndTestFixture
+        public class TestFixture(IMessageSink sink)
+            : EndToEndTestFixture(rootPath, "node", RpcWorkerConstants.NodeLanguageWorkerName)
         {
             private static readonly string rootPath = Path.Combine("TestScripts", "Node");
+            private readonly AzuriteFixture _azurite = new(sink);
```

Comment:
Am I correct in understanding that only the script host is now using the emulator, but the test itself is still using a real storage account via `TestInputContainer` - `ScriptHostEndToEndTestFixture` ? Do we want to change that too or is that out of scope of this PR?

Reply:
Out of scope for this PR. `ScriptHostEndToEndTestFixture` is not used by the immediate tests I am fixing here (`NodeEndToEndTests`). We will update that fixture when we migrate a test using it to the emulator.

Reply:
Oh, that should already be covered in this change. All of the blob, table, and queue clients/containers are derived from the same `CloudStorageAccount`, which will be the emulator here.